web-ts/src/services/BrowserLanguageDetector.ts

```diff
@@ -9,32 +11,33 @@ const localStorageAvailable = () => {
 
   try {
     const localStorage = window.localStorage
-    hasLocalStorageSupport = window !== 'undefined' && localStorage !== null
+    hasLocalStorageSupport = localStorage !== null
     const testKey = 'i18next.translate.boo'
     localStorage.setItem(testKey, 'foo')
     localStorage.removeItem(testKey)
   } catch (e) {
     hasLocalStorageSupport = false
   }
+
   return hasLocalStorageSupport
 }
 
 const LANGUAGE_LOCAL_STORAGE = 'i18nextLng'
-
-export default {
+const languageDetector: LanguageDetectorModule = {
   type: 'languageDetector',
-  async: false,
```

Comment:
Why is this removed? are you sure we dont need it?

Reply:
@Taggotty

Reply:
Seems correct:
Type definition: https://github.com/i18next/i18next/blob/master/index.d.ts#L785
Only usages: https://github.com/i18next/i18next/blob/2482b6117f7aa6a17ee04d24c4dfc6e46cc2138b/i18next.js#L2281

Reply:
It also works correctly AFAICT